app/Http/routes.php

```diff
@@ -34,11 +34,16 @@
 ]);
 
 Route::get('/developer/myapp/{id}', [
```

Comment:
**Code Quality-Nitpick**
@andela-araimi Please group this routes into a group and give the group a `'prefix' => 'developer'`.